user-guide/Standard_Apps/IDP/Tutorials/IDP_Tutorial_DiscoveryAndProvisioning.md

```diff
@@ -0,0 +1,160 @@
+---
+uid: IDP_Tutorial_DiscoveryAndProvisioning
+keywords: idp tutorial, idp kata
+---
+
+# Getting started with IDP and onboarding new equipment
+
+This tutorial will show you how to install IDP and onboard new equiment.
+
+Expected duration: 20 minutes.
+
+<!-- TODO: Uncomment and fill in with correct info when Kata is released.
+> [!TIP]
+> See also: [Kata #xx: IDP Introduction](https://community.dataminer.services/courses/kata-xx/)
+-->
+
+## Prerequisites
+
+- A DataMiner System using DataMiner 10.3.0-CU0 or higher that is
+
+  - [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud).
+  - running on a Microsoft Windows server.
+
+> [!IMPORTANT]
+> If you are using a DataMiner version lower than 10.4.0 or 10.4.3, install the [IDP Migration](https://community.dataminer.services/download/idp-migration/) package first.
+
+> [!NOTE]
+> If you use a [DaaS system](xref:Creating_a_DMS_in_the_cloud), these prerequisites are automatically met.
+
+## Overview
+
+- [Step 1: Deploy IDP from the Catalog](#step-1-deploy-idp-from-the-catalog)
+- [Step 2: Configure discovery profile](#step-2-configure-discovery-profile)
+- [Step 3: Configure scan range](#step-3-configure-scan-range)
+- [Step 4: Run the discovery](#step-4-run-the-discovery)
+- [Step 5: Provision the discovered element(s)](#step-5-provision-the-discovered-elements)
+
+## Step 1: Deploy IDP from the Catalog
```

Comment:
I believe it makes more sense to refer to the installation guide than to duplicate the information here again. 

"Deploy the IDP package with the default settings by following the steps from https://docs.dataminer.services/user-guide/Standard_Apps/IDP/Installing_DataMiner_IDP.html)"

Reply:
As this is one of the steps that will be shown in the Kata, it makes more sense to have it written out here again which is just a link and saying they need to deploy. This way people don't need to swap between different pages just for doing the installation.

Reply:
In this case, I do believe it would be better to refer to the IDP page. Having duplicate information is a great risk, because it's almost certain that at some point the IDP installation guide will need to be updated, and if then we don't remember to update the info here as well (which is also double work), the tutorial will no longer be correct.

Reply:
Changed it.